File: implementation/scipamato/src/test/java/ch/difty/scipamato/lib/FrozenDateTimeServiceTest.java
package ch.difty.scipamato.lib;

import static org.assertj.core.api.Assertions.assertThat;

import org.junit.Test;

public class FrozenDateTimeServiceTest {

    private final DateTimeService service = new FrozenDateTimeService();

    @Test
    public void gettingDateTime() {
        assertThat(service.getCurrentDateTime().toString()).isEqualTo("2016-12-09T06:02:13");
    }

    @Test
    public void gettingTimestamp() {
        assertThat(service.getCurrentTimestamp().toString()).isEqualTo("2016-12-09 06:02:13.0");
    }

    @Test
    public void gettingCurrentDate() {
        assertThat(service.getCurrentDate().toString()).isEqualTo("2016-12-09");
    }
}

File: common/common-utils/src/test/kotlin/ch/difty/scipamato/common/FrozenDateTimeServiceTest.kt
package ch.difty.scipamato.common

import org.amshove.kluent.shouldBeEqualTo
import org.junit.jupiter.api.Test

internal class FrozenDateTimeServiceTest {

    private val service = FrozenDateTimeService()

    @Test
    fun gettingDateTime() {
        service.currentDateTime.toString() shouldBeEqualTo "2016-12-09T06:02:13"
    }

    @Test
    fun gettingTimestamp() {
        service.currentTimestamp.toString() shouldBeEqualTo "2016-12-09 06:02:13.0"
    }

    @Test
    fun gettingCurrentDate() {
        service.currentDate.toString() shouldBeEqualTo "2016-12-09"
    }
}
